### ISSUE TITLE
Include a contact email address in the welcome email.

### ISSUE THREAD

**Issue by darozak** [Darwin-River/Ex-Machinis#188]
I need to tell players how to contact me if they need help.  I should create a support email address for Darwin River and include it in the welcome emails that players get from the registrar.

**Comment by darozak** [comment]
Started working on this.  Modified registrar email files so that they include link to wiki and personal email in case someone needs help (https://github.com/Darwin-River/Ex-Machinis/commit/f48b5a308016a341919af2ffa2af17882d589f3b).  Unfortunately, when I updated the /var/app/User.php file online the email server stopped working and I can't get it back online even after revering to the old User.php script.   Looking for solution.

**Comment by darozak** [comment]
Fernando took a look and noted that the email_handler log for 25 Feb was really large.  This was the result of the IMAP error reported above.  He then found that the registrar, catchall, and mail registration passwords were empty in the .env file.  After fixing this, he found that the DB connection failed because something had happened to the configuration file.  In the end, he managed to get things back online.

**Comment by darozak** [comment]
I spent a good portion of the day trying to get User.php to return my email address in responses from the registrar.  I finally figured out that Gmail was sending any emails from the registrar to the spam folder if they contained the darwinriver.com address in the body of the email.  Could this be because that darwinriver domain doesn't have an SSL associated with it?  I can't think of any other reason why Gmail would be treating that particular account as spam.
